### DISCUSSION TITLE
希望能增加文件搜索功能

### DISCUSSION THREAD

**Discussion by taobao007** [AlistGo/alist#202] [Ideas]
如题，希望可以添加一个全盘文件搜索功能，谢谢大佬。

**Comment by xhofe** [comment]
借用多个地方的回答：
- #153 
- https://github.com/qkqpttgf/OneManager-php/issues/10
- https://github.com/spencerwooo/onedrive-vercel-index/discussions/151

本地并没有存储目录结构，基本大部分都存储提供的API都是不包含目录结构的，很不好处理，所以暂时不会考虑增加搜索。
如果是刚需，可以考虑使用[PanIndex](https://github.com/libsgh/PanIndex)
